hello-kedro/README.md

```diff
@@ -0,0 +1,7 @@
+# Hello Kedro Example
+
+This is the example code described in the [Hello Kedro](https://github.com/quantumblacklabs/private-kedro/blob/develop/docs/source/02_getting_started/04_hello_kedro.md) documentation. 
+
+<!-- NOTE: LINK TO CHANGE SHORTLY -->
```

Comment:
What is this? Is it possible to remove this note, and apply the change with another PR?